tests/helpers.py

```diff
@@ -452,8 +478,22 @@ def delete_all_cephfilesystem():
         kind=constants.CEPHFILESYSTEM,
         namespace=defaults.ROOK_CLUSTER_NAMESPACE
     )
+    assert CFS.delete(resource_name=fs_name)
+    return True
+
+
+def get_cephfs_name():
+    """
+    Function to retrive CephFS name
+
+    Returns:
+        str: Name of CFS
+    """
+    CFS = ocp.OCP(
+        kind=constants.CEPHFILESYSTEM,
+        namespace=defaults.ROOK_CLUSTER_NAMESPACE
+    )
     result = CFS.get()
     cephfs_dict = result['items']
     for item in cephfs_dict:
-        assert CFS.delete(resource_name=item.get('metadata').get('name'))
-    return True
+        return item.get('metadata').get('name')
```

Comment:
no need for iterating over 1 item.
return result['items'][0].get('metadata').get('name')